lua/lazyvim/plugins/extras/coding/codeium.lua

```diff
@@ -0,0 +1,29 @@
+return {
+
+  -- codeium
+  {
+    "Exafunction/codeium.nvim",
+    cmd = "Codeium",
+    build = ":Codeium Auth",
+    opts = {},
+  },
+  -- codeium cmp source
+  {
+    "nvim-cmp",
+    dependencies = {
+      {
+        "Exafunction/codeium.nvim",
+        opts = {},
+        config = function(_, opts)
+          require("codeium").setup(opts)
+        end,
+      },
```

Comment:
This can probably be removed?

Reply:
japp, done.

Reply:
but you still need `opts = {}` right? :) 
Otherwise setup would not be called

Reply:
But `opt` is there on line 8 already?! Tested it locally with success.

Reply:
Nevermind, I only saw the diff from the github notif. All good! :)